### DISCUSSION TITLE
Local build error, v1.0.185-alpha.5

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#6081] [Help] [answered]
![image](https://user-images.githubusercontent.com/93232582/170636070-fddd89a6-3789-4ac1-9280-8416ca6ef437.png)

lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in '@budibase/server'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Build failed with the latest version today. Could anyone help with this? Or recommend me the most recent stable version :)**

**Comment by melohagan** [comment, marked as answer]
Hi @dannyshin76 

Have you tried a `yarn clean` followed by a `yarn setup` ?

**Reply by dannyshin76** [reply]
Yes, we tried that. Any idea what's wrong?

**Reply by dannyshin76** [reply]
We tried with updated version this week, and build was good :)
